### PR TITLE
Beef up CI

### PR DESCRIPTION
**Pull request by jonas-schievink**
Closes https://github.com/rust-embedded/cortex-m-rt/issues/233
Closes https://github.com/rust-embedded/cortex-m-rt/issues/80

### CODE REVIEW

**Review by therealprof — APPROVED** [review]
LGTM.

bors r+

File: .github/bors.toml [hunk 1]
```diff
@@ -4,5 +4,6 @@ required_approvals = 1
 status = [
   "ci-linux (stable)",
   "ci-linux (1.39.0)",
+  "build-other",
```
Comment (adamgreig) [inline .github/bors.toml:7]: I _think_ this will have to be

```
- build-other (macOS-latest)
- build-other (windows-latest)
```

Comment (adamgreig) [reply .github/bors.toml:7]: Guess we'll find out in a minute!

Comment (therealprof) [reply .github/bors.toml]: Yes, you're correct.

**Comment by jonas-schievink** [comment]
bors r=therealprof

**Review by therealprof — APPROVED** [review]
bors r+

**Pull request merged**